Header header
uint32 object_id
uint32 object_label
geometry_msgs/Point point

